tests/end2end/playwright/pages/project.js

```diff
@@ -213,6 +213,47 @@ export class ProjectPage extends BasePage {
         await this.page.locator('#attribute-layer-list-table').locator(`button[value=${layer}]`).click();
     }
 
+    /**
+     * closeAttributeTable function
+     * Close each layer panel and close attribute table dock
+     */
+    async closeAttributeTable(){
+        const tabsCloseButtons = await this.page.locator('.btn-close-attribute-tab');
+        for (let i = 0; i < await tabsCloseButtons.count(); i ++){
+            await tabsCloseButtons.nth(i).click();
+        }
+
+        await this.page.locator('a#button-attributeLayers').click();
+    }
+
+    /**
+     * launchExport function
+     * @param {string} layer the layer to export
+     * @param {null|string} bbox bounding box if limitDataToBbox is set
+     */
+    async launchExport(layer, bbox = null){
+        const getFeatureRequestPromise = this.page.waitForRequest(
+            request => request.method() === 'POST'
+            && request.postData() != null
+            && request.postData()?.includes('GetFeature') === true
+        );
+        // bbox in getFeature request for export
+        const getFeatureRequestContains = request => {
+            const postData = request.postData();
+            expect(postData).toContain('SERVICE=WFS');
+            expect(postData).toContain('REQUEST=GetFeature');
+            if(bbox) {
+                let bboxParameter = new RegExp(String.raw`BBOX=${bbox}`, "g");
+                expect(postData).toMatch(bboxParameter);
+            }
+        };
+
+        await this.page.getByRole('button', { name: 'Export' }).click();
+        await this.page.getByRole('button', { name: 'GeoJSON' }).click();
+
+        getFeatureRequestContains(await getFeatureRequestPromise);
+    }
```

Comment:
```suggestion
    /**
     * launchExport function
     * @param {string} layer     - the layer to export
     * @param {string} format - the format to export
     * @returns {Promise<Request>} the request generated by the export
     */
    async launchExport(layer, format){
        const getFeatureRequestPromise = this.page.waitForRequest(
            request => request.method() === 'POST'
            && request.postData() != null
            && request.postData()?.includes('GetFeature') === true
        );

        await this.page.getByRole('button', { name: 'Export' }).click();
        await this.page.getByRole('button', { name: format }).click();

        return await getFeatureRequestPromise;
    }
```
I am in favor to do test outside POM.